GCC 4.4.7 cc1 segfault on centos6_64 (vagrant VM)

when trying current master branch on centos6 64bit:
```
==> centos6_64: Successfully installed Cython-0.25.2 apipkg-1.4 coverage-4.4.1 execnet-1.4.1 pluggy-0.4.0 py-1.4.34 pytest-3.1.1 pytest-benchmark-3.0.0 pytest-cov-2.5.1 pytest-xdist-1.16.0 setuptools-scm-1.15.5 tox-2.7.0 virtualenv-15.1.0
==> centos6_64: running clean
==> centos6_64: removing 'build/temp.linux-x86_64-3.5' (and everything under it)
==> centos6_64: removing 'build'
==> centos6_64: rm src/borg/compress.c
==> centos6_64: rm src/borg/compress.cpython-35m-x86_64-linux-gnu.so
==> centos6_64: rm src/borg/crypto/low_level.c
==> centos6_64: rm src/borg/crypto/low_level.cpython-35m-x86_64-linux-gnu.so
==> centos6_64: rm src/borg/algorithms/chunker.c
==> centos6_64: rm src/borg/algorithms/chunker.cpython-35m-x86_64-linux-gnu.so
==> centos6_64: rm src/borg/hashindex.c
==> centos6_64: rm src/borg/hashindex.cpython-35m-x86_64-linux-gnu.so
==> centos6_64: rm src/borg/item.c
==> centos6_64: rm src/borg/item.cpython-35m-x86_64-linux-gnu.so
==> centos6_64: rm src/borg/algorithms/checksums.c
==> centos6_64: rm src/borg/algorithms/checksums.cpython-35m-x86_64-linux-gnu.so
==> centos6_64: rm src/borg/platform/posix.c
==> centos6_64: rm src/borg/platform/posix.cpython-35m-x86_64-linux-gnu.so
==> centos6_64: rm src/borg/platform/linux.c
==> centos6_64: rm src/borg/platform/linux.cpython-35m-x86_64-linux-gnu.so
==> centos6_64: Obtaining file:///vagrant/borg/borg
==> centos6_64: Collecting msgpack-python>=0.4.6 (from borgbackup==1.0.11.dev83+g6a929c6)
==> centos6_64:   Downloading msgpack-python-0.4.8.tar.gz (113kB)
==> centos6_64: Building wheels for collected packages: msgpack-python
==> centos6_64:   Running setup.py bdist_wheel for msgpack-python: started
==> centos6_64:   Running setup.py bdist_wheel for msgpack-python: finished with status 'done'
==> centos6_64:   Stored in directory: /home/vagrant/.cache/pip/wheels/2c/e7/e7/9031652a69d594665c5ca25e41d0fb3faa024e730b590e4402
==> centos6_64: Successfully built msgpack-python
==> centos6_64: Installing collected packages: msgpack-python, borgbackup
==> centos6_64:   Running setup.py develop for borgbackup
==> centos6_64:     Complete output from command /home/vagrant/.pyenv/versions/3.5.3/envs/borg-env/bin/python -c "import setuptools, tokenize;__file__='/vagrant/borg/borg/setup.py';f=getattr(tokenize, 'open', open)(__file__);code=f.read().replace('\r\n', '\n');f.close();exec(compile(code, __file__, 'exec'))" develop --no-deps:
==> centos6_64:     running develop
==> centos6_64:     running egg_info
==> centos6_64:     writing top-level names to src/borgbackup.egg-info/top_level.txt
==> centos6_64:     writing requirements to src/borgbackup.egg-info/requires.txt
==> centos6_64:     writing entry points to src/borgbackup.egg-info/entry_points.txt
==> centos6_64:     writing dependency_links to src/borgbackup.egg-info/dependency_links.txt
==> centos6_64:     writing src/borgbackup.egg-info/PKG-INFO
==> centos6_64:     warning: manifest_maker: standard file '-c' not found
==> centos6_64:     
==> centos6_64:     reading manifest template 'MANIFEST.in'
==> centos6_64:     warning: no previously-included files matching '*.dll' found anywhere in distribution
==> centos6_64:     writing manifest file 'src/borgbackup.egg-info/SOURCES.txt'
==> centos6_64:     running build_ext
==> centos6_64:     cythoning src/borg/compress.pyx to src/borg/compress.c
==> centos6_64:     building 'borg.compress' extension
==> centos6_64:     creating build
==> centos6_64:     creating build/temp.linux-x86_64-3.5
==> centos6_64:     creating build/temp.linux-x86_64-3.5/src
==> centos6_64:     creating build/temp.linux-x86_64-3.5/src/borg
==> centos6_64:     gcc -pthread -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall -Wstrict-prototypes -fPIC -I/usr/include -I/usr/include -I/home/vagrant/.pyenv/versions/3.5.3/envs/borg-env/include -I/home/vagrant/.pyenv/versions/3.5.3/include/python3.5m -c src/borg/compress.c -o build/temp.linux-x86_64-3.5/src/borg/compress.o
==> centos6_64:     gcc -pthread -shared -L/home/vagrant/.pyenv/versions/3.5.3/lib -Wl,-rpath=/home/vagrant/.pyenv/versions/3.5.3/lib build/temp.linux-x86_64-3.5/src/borg/compress.o -L/usr/lib -L/usr/lib -L/home/vagrant/.pyenv/versions/3.5.3/lib -llz4 -lpython3.5m -o /vagrant/borg/borg/src/borg/compress.cpython-35m-x86_64-linux-gnu.so
==> centos6_64:     cythoning src/borg/crypto/low_level.pyx to src/borg/crypto/low_level.c
==> centos6_64:     building 'borg.crypto.low_level' extension
==> centos6_64:     creating build/temp.linux-x86_64-3.5/src/borg/crypto
==> centos6_64:     gcc -pthread -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall -Wstrict-prototypes -fPIC -I/usr/include -I/usr/include -I/home/vagrant/.pyenv/versions/3.5.3/envs/borg-env/include -I/home/vagrant/.pyenv/versions/3.5.3/include/python3.5m -c src/borg/crypto/low_level.c -o build/temp.linux-x86_64-3.5/src/borg/crypto/low_level.o
==> centos6_64:     src/borg/crypto/low_level.c: In function ‘__pyx_pf_4borg_6crypto_9low_level_3AES_8decrypt’:
==> centos6_64:     src/borg/crypto/low_level.c:2385: warning: ‘__pyx_r$buf’ may be used uninitialized in this function
==> centos6_64:     src/borg/crypto/low_level.c:2385: note: ‘__pyx_r$buf’ was declared here
==> centos6_64:     src/borg/crypto/low_level.c:2385: warning: ‘__pyx_r$obj’ may be used uninitialized in this function
==> centos6_64:     src/borg/crypto/low_level.c:2385: note: ‘__pyx_r$obj’ was declared here
==> centos6_64:     src/borg/crypto/low_level.c:2385: warning: ‘__pyx_r$len’ may be used uninitialized in this function
==> centos6_64:     src/borg/crypto/low_level.c:2385: note: ‘__pyx_r$len’ was declared here
==> centos6_64:     src/borg/crypto/low_level.c:2385: warning: ‘__pyx_r$itemsize’ may be used uninitialized in this function
==> centos6_64:     src/borg/crypto/low_level.c:2385: note: ‘__pyx_r$itemsize’ was declared here
==> centos6_64:     src/borg/crypto/low_level.c:2385: warning: ‘__pyx_r$format’ may be used uninitialized in this function
==> centos6_64:     src/borg/crypto/low_level.c:2385: note: ‘__pyx_r$format’ was declared here
==> centos6_64:     src/borg/crypto/low_level.c:2385: warning: ‘__pyx_r$shape’ may be used uninitialized in this function
==> centos6_64:     src/borg/crypto/low_level.c:2385: note: ‘__pyx_r$shape’ was declared here
==> centos6_64:     src/borg/crypto/low_level.c:2385: warning: ‘__pyx_r$strides’ may be used uninitialized in this function
==> centos6_64:     src/borg/crypto/low_level.c:2385: note: ‘__pyx_r$strides’ was declared here
==> centos6_64:     src/borg/crypto/low_level.c:2385: warning: ‘__pyx_r$suboffsets’ may be used uninitialized in this function
==> centos6_64:     src/borg/crypto/low_level.c:2385: note: ‘__pyx_r$suboffsets’ was declared here
==> centos6_64:     src/borg/crypto/low_level.c:2385: warning: ‘__pyx_r$internal’ may be used uninitialized in this function
==> centos6_64:     src/borg/crypto/low_level.c:2385: note: ‘__pyx_r$internal’ was declared here
==> centos6_64:     src/borg/crypto/low_level.c: In function ‘__pyx_pf_4borg_6crypto_9low_level_3AES_6encrypt’:
==> centos6_64:     src/borg/crypto/low_level.c:2385: warning: ‘__pyx_r$buf’ may be used uninitialized in this function
==> centos6_64:     src/borg/crypto/low_level.c:2385: note: ‘__pyx_r$buf’ was declared here
==> centos6_64:     src/borg/crypto/low_level.c:2385: warning: ‘__pyx_r$obj’ may be used uninitialized in this function
==> centos6_64:     src/borg/crypto/low_level.c:2385: note: ‘__pyx_r$obj’ was declared here
==> centos6_64:     src/borg/crypto/low_level.c:2385: warning: ‘__pyx_r$len’ may be used uninitialized in this function
==> centos6_64:     src/borg/crypto/low_level.c:2385: note: ‘__pyx_r$len’ was declared here
==> centos6_64:     src/borg/crypto/low_level.c:2385: warning: ‘__pyx_r$itemsize’ may be used uninitialized in this function
==> centos6_64:     src/borg/crypto/low_level.c:2385: note: ‘__pyx_r$itemsize’ was declared here
==> centos6_64:     src/borg/crypto/low_level.c:2385: warning: ‘__pyx_r$format’ may be used uninitialized in this function
==> centos6_64:     src/borg/crypto/low_level.c:2385: note: ‘__pyx_r$format’ was declared here
==> centos6_64:     src/borg/crypto/low_level.c:2385: warning: ‘__pyx_r$shape’ may be used uninitialized in this function
==> centos6_64:     src/borg/crypto/low_level.c:2385: note: ‘__pyx_r$shape’ was declared here
==> centos6_64:     src/borg/crypto/low_level.c:2385: warning: ‘__pyx_r$strides’ may be used uninitialized in this function
==> centos6_64:     src/borg/crypto/low_level.c:2385: note: ‘__pyx_r$strides’ was declared here
==> centos6_64:     src/borg/crypto/low_level.c:2385: warning: ‘__pyx_r$suboffsets’ may be used uninitialized in this function
==> centos6_64:     src/borg/crypto/low_level.c:2385: note: ‘__pyx_r$suboffsets’ was declared here
==> centos6_64:     src/borg/crypto/low_level.c:2385: warning: ‘__pyx_r$internal’ may be used uninitialized in this function
==> centos6_64:     src/borg/crypto/low_level.c:2385: note: ‘__pyx_r$internal’ was declared here
==> centos6_64:     gcc -pthread -shared -L/home/vagrant/.pyenv/versions/3.5.3/lib -Wl,-rpath=/home/vagrant/.pyenv/versions/3.5.3/lib build/temp.linux-x86_64-3.5/src/borg/crypto/low_level.o -L/usr/lib -L/usr/lib -L/home/vagrant/.pyenv/versions/3.5.3/lib -lcrypto -lpython3.5m -o /vagrant/borg/borg/src/borg/crypto/low_level.cpython-35m-x86_64-linux-gnu.so
==> centos6_64:     cythoning src/borg/hashindex.pyx to src/borg/hashindex.c
==> centos6_64:     building 'borg.hashindex' extension
==> centos6_64:     gcc -pthread -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall -Wstrict-prototypes -fPIC -I/home/vagrant/.pyenv/versions/3.5.3/envs/borg-env/include -I/home/vagrant/.pyenv/versions/3.5.3/include/python3.5m -c src/borg/hashindex.c -o build/temp.linux-x86_64-3.5/src/borg/hashindex.o
==> centos6_64:     In file included from src/borg/hashindex.c:436:
==> centos6_64:     src/borg/_hashindex.c: In function ‘hashindex_set’:
==> centos6_64:     src/borg/_hashindex.c:538: warning: ‘start_idx’ may be used uninitialized in this function
==> centos6_64:     gcc -pthread -shared -L/home/vagrant/.pyenv/versions/3.5.3/lib -Wl,-rpath=/home/vagrant/.pyenv/versions/3.5.3/lib build/temp.linux-x86_64-3.5/src/borg/hashindex.o -L/home/vagrant/.pyenv/versions/3.5.3/lib -lpython3.5m -o /vagrant/borg/borg/src/borg/hashindex.cpython-35m-x86_64-linux-gnu.so
==> centos6_64:     cythoning src/borg/item.pyx to src/borg/item.c
==> centos6_64:     building 'borg.item' extension
==> centos6_64:     gcc -pthread -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall -Wstrict-prototypes -fPIC -I/home/vagrant/.pyenv/versions/3.5.3/envs/borg-env/include -I/home/vagrant/.pyenv/versions/3.5.3/include/python3.5m -c src/borg/item.c -o build/temp.linux-x86_64-3.5/src/borg/item.o
==> centos6_64:     gcc -pthread -shared -L/home/vagrant/.pyenv/versions/3.5.3/lib -Wl,-rpath=/home/vagrant/.pyenv/versions/3.5.3/lib build/temp.linux-x86_64-3.5/src/borg/item.o -L/home/vagrant/.pyenv/versions/3.5.3/lib -lpython3.5m -o /vagrant/borg/borg/src/borg/item.cpython-35m-x86_64-linux-gnu.so
==> centos6_64:     cythoning src/borg/algorithms/chunker.pyx to src/borg/algorithms/chunker.c
==> centos6_64:     building 'borg.algorithms.chunker' extension
==> centos6_64:     creating build/temp.linux-x86_64-3.5/src/borg/algorithms
==> centos6_64:     gcc -pthread -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall -Wstrict-prototypes -fPIC -I/home/vagrant/.pyenv/versions/3.5.3/envs/borg-env/include -I/home/vagrant/.pyenv/versions/3.5.3/include/python3.5m -c src/borg/algorithms/chunker.c -o build/temp.linux-x86_64-3.5/src/borg/algorithms/chunker.o
==> centos6_64:     gcc -pthread -shared -L/home/vagrant/.pyenv/versions/3.5.3/lib -Wl,-rpath=/home/vagrant/.pyenv/versions/3.5.3/lib build/temp.linux-x86_64-3.5/src/borg/algorithms/chunker.o -L/home/vagrant/.pyenv/versions/3.5.3/lib -lpython3.5m -o /vagrant/borg/borg/src/borg/algorithms/chunker.cpython-35m-x86_64-linux-gnu.so
==> centos6_64:     cythoning src/borg/algorithms/checksums.pyx to src/borg/algorithms/checksums.c
==> centos6_64:     building 'borg.algorithms.checksums' extension
==> centos6_64:     gcc -pthread -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall -Wstrict-prototypes -fPIC -I/home/vagrant/.pyenv/versions/3.5.3/envs/borg-env/include -I/home/vagrant/.pyenv/versions/3.5.3/include/python3.5m -c src/borg/algorithms/checksums.c -o build/temp.linux-x86_64-3.5/src/borg/algorithms/checksums.o
==> centos6_64:     gcc: Internal error: Segmentation fault (program cc1)
==> centos6_64:     Please submit a full bug report.
==> centos6_64:     See <http://bugzilla.redhat.com/bugzilla> for instructions.
==> centos6_64:     error: command 'gcc' failed with exit status 1


==> centos6_64:     
==> centos6_64:     ----------------------------------------
==> centos6_64: Command "/home/vagrant/.pyenv/versions/3.5.3/envs/borg-env/bin/python -c "import setuptools, tokenize;__file__='/vagrant/borg/borg/setup.py';f=getattr(tokenize, 'open', open)(__file__);code=f.read().replace('\r\n', '\n');f.close();exec(compile(code, __file__, 'exec'))" develop --no-deps" failed with error code 1 in /vagrant/borg/borg/
==> centos6_64: Running provisioner: run tests (shell)...
    centos6_64: Running: inline script
==> centos6_64: /home/vagrant/.pyenv/bin/pyenv
==> centos6_64: /usr/bin/fakeroot
==> centos6_64: Running tox WITH fakeroot -u
==> centos6_64: GLOB sdist-make: /vagrant/borg/borg/setup.py
==> centos6_64: py34 recreate: /vagrant/borg/borg/.tox/py34
==> centos6_64: py34 installdeps: -rrequirements.d/development.txt, -rrequirements.d/attic.txt
==> centos6_64: py34 inst: /vagrant/borg/borg/.tox/dist/borgbackup-1.1.0b6.dev193+g71495b9.d20170605.zip
==> centos6_64: ERROR: invocation failed (exit code 1), logfile: /vagrant/borg/borg/.tox/py34/log/py34-2.log
==> centos6_64: ERROR: actionid: py34
==> centos6_64: msg: installpkg
==> centos6_64: cmdargs: ['/vagrant/borg/borg/.tox/py34/bin/pip', 'install', '/vagrant/borg/borg/.tox/dist/borgbackup-1.1.0b6.dev193+g71495b9.d20170605.zip']
==> centos6_64: env: {'_': '/home/vagrant/.pyenv/versions/3.5.3/envs/borg-env/bin/tox', 'TERM': 'vt100', 'PATH': '/vagrant/borg/borg/.tox/py34/bin:/home/vagrant/.pyenv/versions/3.5.3/envs/borg-env/bin:/home/vagrant/.pyenv/plugins/pyenv-virtualenv/shims:/home/vagrant/.pyenv/shims:/home/vagrant/.pyenv/bin:/vagrant/borg:/home/vagrant/.pyenv/plugins/pyenv-virtualenv/shims:/home/vagrant/.pyenv/shims:/home/vagrant/.pyenv/bin:/vagrant/borg:/usr/local/bin:/bin:/usr/bin:/usr/local/sbin:/usr/sbin:/sbin:/home/vagrant/bin:/home/vagrant/bin', 'HOSTNAME': 'centos6-64', 'LD_PRELOAD': 'libfakeroot-tcp.so', 'USER': 'vagrant', 'SSH_CLIENT': '10.0.2.2 48362 22', 'HISTCONTROL': 'ignoredups', 'PYTHON_CONFIGURE_OPTS': '--enable-shared', 'SELINUX_USE_CURRENT_RANGE': '', 'PYENV_VIRTUALENV_INIT': '1', 'PYTHONHASHSEED': '3172688560', 'VIRTUAL_ENV': '/vagrant/borg/borg/.tox/py34', 'PYENV_SHELL': 'bash', 'HISTSIZE': '1000', 'PWD': '/vagrant/borg/borg', 'LESSOPEN': '||/usr/bin/lesspipe.sh %s', 'G_BROKEN_FILENAMES': '1', 'SHELL': '/bin/bash', 'FAKEROOTKEY': '38651', 'LANG': 'en_US.UTF-8', 'MAIL': '/var/spool/mail/vagrant', 'LOGNAME': 'vagrant', 'LD_LIBRARY_PATH': '/usr/lib64/libfakeroot:/usr/lib64/libfakeroot:/usr/lib32/libfakeroot', 'FAKED_MODE': 'unknown-is-real', 'SELINUX_ROLE_REQUESTED': '', 'SSH_CONNECTION': '10.0.2.2 48362 10.0.2.15 22', 'SHLVL': '4', 'SELINUX_LEVEL_REQUESTED': '', 'HOME': '/home/vagrant', 'XDISTN': '12'}
==> centos6_64: 
==> centos6_64: The directory '/home/vagrant/.cache/pip/http' or its parent directory is not owned by the current user and the cache has been disabled. Please check the permissions and owner of that directory. If executing pip with sudo, you may want sudo's -H flag.
==> centos6_64: The directory '/home/vagrant/.cache/pip' or its parent directory is not owned by the current user and caching wheels has been disabled. check the permissions and owner of that directory. If executing pip with sudo, you may want sudo's -H flag.
==> centos6_64: Processing ./.tox/dist/borgbackup-1.1.0b6.dev193+g71495b9.d20170605.zip
==> centos6_64: Requirement already satisfied: msgpack-python>=0.4.6 in ./.tox/py34/lib/python3.4/site-packages (from borgbackup==1.1.0b6.dev193+g71495b9.d20170605)
==> centos6_64: Installing collected packages: borgbackup
==> centos6_64:   Running setup.py install for borgbackup: started
==> centos6_64:     Running setup.py install for borgbackup: finished with status 'error'
==> centos6_64:     Complete output from command /vagrant/borg/borg/.tox/py34/bin/python3.4 -u -c "import setuptools, tokenize;__file__='/tmp/pip-6ocpyvmj-build/setup.py';f=getattr(tokenize, 'open', open)(__file__);code=f.read().replace('\r\n', '\n');f.close();exec(compile(code, __file__, 'exec'))" install --record /tmp/pip-f4p6krlg-record/install-record.txt --single-version-externally-managed --compile --install-headers /vagrant/borg/borg/.tox/py34/include/site/python3.4/borgbackup:
==> centos6_64:     running install
==> centos6_64:     running build
==> centos6_64:     running build_py
==> centos6_64:     creating build
==> centos6_64:     creating build/lib.linux-x86_64-3.4
==> centos6_64:     creating build/lib.linux-x86_64-3.4/borg
==> centos6_64:     copying src/borg/cache.py -> build/lib.linux-x86_64-3.4/borg
...
==> centos6_64:     creating build/lib.linux-x86_64-3.4/borg/algorithms
==> centos6_64:     copying src/borg/algorithms/__init__.py -> build/lib.linux-x86_64-3.4/borg/algorithms
==> centos6_64:     running egg_info
==> centos6_64:     writing requirements to src/borgbackup.egg-info/requires.txt
==> centos6_64:     writing src/borgbackup.egg-info/PKG-INFO
==> centos6_64:     writing dependency_links to src/borgbackup.egg-info/dependency_links.txt
==> centos6_64:     writing entry points to src/borgbackup.egg-info/entry_points.txt
==> centos6_64:     writing top-level names to src/borgbackup.egg-info/top_level.txt
==> centos6_64:     reading manifest file 'src/borgbackup.egg-info/SOURCES.txt'
==> centos6_64:     reading manifest template 'MANIFEST.in'
==> centos6_64:     writing manifest file 'src/borgbackup.egg-info/SOURCES.txt'
==> centos6_64:     copying src/borg/_hashindex.c -> build/lib.linux-x86_64-3.4/borg
...
==> centos6_64:     creating build/lib.linux-x86_64-3.4/borg/algorithms/blake2
==> centos6_64:     copying src/borg/algorithms/blake2/COPYING -> build/lib.linux-x86_64-3.4/borg/algorithms/blake2
...
==> centos6_64:     running build_ext
==> centos6_64:     skipping 'src/borg/compress.c' Cython extension (up-to-date)
==> centos6_64:     building 'borg.compress' extension
==> centos6_64:     creating build/temp.linux-x86_64-3.4
==> centos6_64:     creating build/temp.linux-x86_64-3.4/src
==> centos6_64:     creating build/temp.linux-x86_64-3.4/src/borg
==> centos6_64:     gcc -pthread -Werror=declaration-after-statement -DNDEBUG -g -fwrapv -O3 -Wall -Wstrict-prototypes -fPIC -I/usr/include -I/usr/include -I/home/vagrant/.pyenv/versions/3.4.0/include/python3.4m -c src/borg/compress.c -o build/temp.linux-x86_64-3.4/src/borg/compress.o
==> centos6_64:     gcc -pthread -shared -L/home/vagrant/.pyenv/versions/3.4.0/lib -Wl,-rpath=/home/vagrant/.pyenv/versions/3.4.0/lib build/temp.linux-x86_64-3.4/src/borg/compress.o -L/usr/lib -L/usr/lib -L/home/vagrant/.pyenv/versions/3.4.0/lib -llz4 -lpython3.4m -o build/lib.linux-x86_64-3.4/borg/compress.cpython-34m.so
==> centos6_64:     skipping 'src/borg/crypto/low_level.c' Cython extension (up-to-date)
==> centos6_64:     building 'borg.crypto.low_level' extension
==> centos6_64:     creating build/temp.linux-x86_64-3.4/src/borg/crypto
==> centos6_64:     gcc -pthread -Werror=declaration-after-statement -DNDEBUG -g -fwrapv -O3 -Wall -Wstrict-prototypes -fPIC -I/usr/include -I/usr/include -I/home/vagrant/.pyenv/versions/3.4.0/include/python3.4m -c src/borg/crypto/low_level.c -o build/temp.linux-x86_64-3.4/src/borg/crypto/low_level.o
==> centos6_64:     src/borg/crypto/low_level.c: In function ‘__pyx_pf_4borg_6crypto_9low_level_3AES_8decrypt’:
==> centos6_64:     src/borg/crypto/low_level.c:2385: warning: ‘__pyx_r$buf’ may be used uninitialized in this function
==> centos6_64:     src/borg/crypto/low_level.c:2385: note: ‘__pyx_r$buf’ was declared here
==> centos6_64:     src/borg/crypto/low_level.c:2385: warning: ‘__pyx_r$obj’ may be used uninitialized in this function
==> centos6_64:     src/borg/crypto/low_level.c:2385: note: ‘__pyx_r$obj’ was declared here
==> centos6_64:     src/borg/crypto/low_level.c:2385: warning: ‘__pyx_r$len’ may be used uninitialized in this function
==> centos6_64:     src/borg/crypto/low_level.c:2385: note: ‘__pyx_r$len’ was declared here
==> centos6_64:     src/borg/crypto/low_level.c:2385: warning: ‘__pyx_r$itemsize’ may be used uninitialized in this function
==> centos6_64:     src/borg/crypto/low_level.c:2385: note: ‘__pyx_r$itemsize’ was declared here
==> centos6_64:     src/borg/crypto/low_level.c:2385: warning: ‘__pyx_r$format’ may be used uninitialized in this function
==> centos6_64:     src/borg/crypto/low_level.c:2385: note: ‘__pyx_r$format’ was declared here
==> centos6_64:     src/borg/crypto/low_level.c:2385: warning: ‘__pyx_r$shape’ may be used uninitialized in this function
==> centos6_64:     src/borg/crypto/low_level.c:2385: note: ‘__pyx_r$shape’ was declared here
==> centos6_64:     src/borg/crypto/low_level.c:2385: warning: ‘__pyx_r$strides’ may be used uninitialized in this function
==> centos6_64:     src/borg/crypto/low_level.c:2385: note: ‘__pyx_r$strides’ was declared here
==> centos6_64:     src/borg/crypto/low_level.c:2385: warning: ‘__pyx_r$suboffsets’ may be used uninitialized in this function
==> centos6_64:     src/borg/crypto/low_level.c:2385: note: ‘__pyx_r$suboffsets’ was declared here
==> centos6_64:     src/borg/crypto/low_level.c:2385: warning: ‘__pyx_r$internal’ may be used uninitialized in this function
==> centos6_64:     src/borg/crypto/low_level.c:2385: note: ‘__pyx_r$internal’ was declared here
==> centos6_64:     src/borg/crypto/low_level.c: In function ‘__pyx_pf_4borg_6crypto_9low_level_3AES_6encrypt’:
==> centos6_64:     src/borg/crypto/low_level.c:2385: warning: ‘__pyx_r$buf’ may be used uninitialized in this function
==> centos6_64:     src/borg/crypto/low_level.c:2385: note: ‘__pyx_r$buf’ was declared here
==> centos6_64:     src/borg/crypto/low_level.c:2385: warning: ‘__pyx_r$obj’ may be used uninitialized in this function
==> centos6_64:     src/borg/crypto/low_level.c:2385: note: ‘__pyx_r$obj’ was declared here
==> centos6_64:     src/borg/crypto/low_level.c:2385: warning: ‘__pyx_r$len’ may be used uninitialized in this function
==> centos6_64:     src/borg/crypto/low_level.c:2385: note: ‘__pyx_r$len’ was declared here
==> centos6_64:     src/borg/crypto/low_level.c:2385: warning: ‘__pyx_r$itemsize’ may be used uninitialized in this function
==> centos6_64:     src/borg/crypto/low_level.c:2385: note: ‘__pyx_r$itemsize’ was declared here
==> centos6_64:     src/borg/crypto/low_level.c:2385: warning: ‘__pyx_r$format’ may be used uninitialized in this function
==> centos6_64:     src/borg/crypto/low_level.c:2385: note: ‘__pyx_r$format’ was declared here
==> centos6_64:     src/borg/crypto/low_level.c:2385: warning: ‘__pyx_r$shape’ may be used uni
==> centos6_64: nitialized in this function
==> centos6_64:     src/borg/crypto/low_level.c:2385: note: ‘__pyx_r$shape’ was declared here
==> centos6_64:     src/borg/crypto/low_level.c:2385: warning: ‘__pyx_r$strides’ may be used uninitialized in this function
==> centos6_64:     src/borg/crypto/low_level.c:2385: note: ‘__pyx_r$strides’ was declared here
==> centos6_64:     src/borg/crypto/low_level.c:2385: warning: ‘__pyx_r$suboffsets’ may be used uninitialized in this function
==> centos6_64:     src/borg/crypto/low_level.c:2385: note: ‘__pyx_r$suboffsets’ was declared here
==> centos6_64:     src/borg/crypto/low_level.c:2385: warning: ‘__pyx_r$internal’ may be used uninitialized in this function
==> centos6_64:     src/borg/crypto/low_level.c:2385: note: ‘__pyx_r$internal’ was declared here
==> centos6_64:     gcc -pthread -shared -L/home/vagrant/.pyenv/versions/3.4.0/lib -Wl,-rpath=/home/vagrant/.pyenv/versions/3.4.0/lib build/temp.linux-x86_64-3.4/src/borg/crypto/low_level.o -L/usr/lib -L/usr/lib -L/home/vagrant/.pyenv/versions/3.4.0/lib -lcrypto -lpython3.4m -o build/lib.linux-x86_64-3.4/borg/crypto/low_level.cpython-34m.so
==> centos6_64:     skipping 'src/borg/hashindex.c' Cython extension (up-to-date)
==> centos6_64:     building 'borg.hashindex' extension
==> centos6_64:     gcc -pthread -Werror=declaration-after-statement -DNDEBUG -g -fwrapv -O3 -Wall -Wstrict-prototypes -fPIC -I/home/vagrant/.pyenv/versions/3.4.0/include/python3.4m -c src/borg/hashindex.c -o build/temp.linux-x86_64-3.4/src/borg/hashindex.o
==> centos6_64:     In file included from src/borg/hashindex.c:436:
==> centos6_64:     src/borg/_hashindex.c: In function ‘hashindex_set’:
==> centos6_64:     src/borg/_hashindex.c:538: warning: ‘start_idx’ may be used uninitialized in this function
==> centos6_64:     gcc -pthread -shared -L/home/vagrant/.pyenv/versions/3.4.0/lib -Wl,-rpath=/home/vagrant/.pyenv/versions/3.4.0/lib build/temp.linux-x86_64-3.4/src/borg/hashindex.o -L/home/vagrant/.pyenv/versions/3.4.0/lib -lpython3.4m -o build/lib.linux-x86_64-3.4/borg/hashindex.cpython-34m.so
==> centos6_64:     skipping 'src/borg/item.c' Cython extension (up-to-date)
==> centos6_64:     building 'borg.item' extension
==> centos6_64:     gcc -pthread -Werror=declaration-after-statement -DNDEBUG -g -fwrapv -O3 -Wall -Wstrict-prototypes -fPIC -I/home/vagrant/.pyenv/versions/3.4.0/include/python3.4m -c src/borg/item.c -o build/temp.linux-x86_64-3.4/src/borg/item.o
==> centos6_64:     gcc -pthread -shared -L/home/vagrant/.pyenv/versions/3.4.0/lib -Wl,-rpath=/home/vagrant/.pyenv/versions/3.4.0/lib build/temp.linux-x86_64-3.4/src/borg/item.o -L/home/vagrant/.pyenv/versions/3.4.0/lib -lpython3.4m -o build/lib.linux-x86_64-3.4/borg/item.cpython-34m.so
==> centos6_64:     skipping 'src/borg/algorithms/chunker.c' Cython extension (up-to-date)
==> centos6_64:     building 'borg.algorithms.chunker' extension
==> centos6_64:     creating build/temp.linux-x86_64-3.4/src/borg/algorithms
==> centos6_64:     gcc -pthread -Werror=declaration-after-statement -DNDEBUG -g -fwrapv -O3 -Wall -Wstrict-prototypes -fPIC -I/home/vagrant/.pyenv/versions/3.4.0/include/python3.4m -c src/borg/algorithms/chunker.c -o build/temp.linux-x86_64-3.4/src/borg/algorithms/chunker.o
==> centos6_64:     gcc -pthread -shared -L/home/vagrant/.pyenv/versions/3.4.0/lib -Wl,-rpath=/home/vagrant/.pyenv/versions/3.4.0/lib build/temp.linux-x86_64-3.4/src/borg/algorithms/chunker.o -L/home/vagrant/.pyenv/versions/3.4.0/lib -lpython3.4m -o build/lib.linux-x86_64-3.4/borg/algorithms/chunker.cpython-34m.so
==> centos6_64:     skipping 'src/borg/algorithms/checksums.c' Cython extension (up-to-date)
==> centos6_64:     building 'borg.algorithms.checksums' extension
==> centos6_64:     gcc -pthread -Werror=declaration-after-statement -DNDEBUG -g -fwrapv -O3 -Wall -Wstrict-prototypes -fPIC -I/home/vagrant/.pyenv/versions/3.4.0/include/python3.4m -c src/borg/algorithms/checksums.c -o build/temp.linux-x86_64-3.4/src/borg/algorithms/checksums.o
==> centos6_64:     gcc: Internal error: Segmentation fault (program cc1)
==> centos6_64:     Please submit a full bug report.
==> centos6_64:     See <http://bugzilla.redhat.com/bugzilla> for instructions.
==> centos6_64:     error: command 'gcc' failed with exit status 1
==> centos6_64:     
==> centos6_64:     ----------------------------------------
==> centos6_64: Command "/vagrant/borg/borg/.tox/py34/bin/python3.4 -u -c "import setuptools, tokenize;__file__='/tmp/pip-6ocpyvmj-build/setup.py';f=getattr(tokenize, 'open', open)(__file__);code=f.read().replace('\r\n', '\n');f.close();exec(compile(code, __file__, 'exec'))" install --record /tmp/pip-f4p6krlg-record/install-record.txt --single-version-externally-managed --compile --install-headers /vagrant/borg/borg/.tox/py34/include/site/python3.4/borgbackup" failed with error code 1 in /tmp/pip-6ocpyvmj-build/
==> centos6_64: py35 recreate: /vagrant/borg/borg/.tox/py35
==> centos6_64: py35 installdeps: -rrequirements.d/development.txt, -rrequirements.d/attic.txt
==> centos6_64: py35 inst: /vagrant/borg/borg/.tox/dist/borgbackup-1.1.0b6.dev193+g71495b9.d20170605.zip
==> centos6_64: ERROR: invocation failed (exit code 1), logfile: /vagrant/borg/borg/.tox/py35/log/py35-2.log
==> centos6_64: ERROR: actionid: py35
==> centos6_64: msg: installpkg
==> centos6_64: cmdargs: ['/vagrant/borg/borg/.tox/py35/bin/pip', 'install', '/vagrant/borg/borg/.tox/dist/borgbackup-1.1.0b6.dev193+g71495b9.d20170605.zip']
==> centos6_64: env: {'_': '/home/vagrant/.pyenv/versions/3.5.3/envs/borg-env/bin/tox', 'TERM': 'vt100', 'PATH': '/vagrant/borg/borg/.tox/py35/bin:/home/vagrant/.pyenv/versions/3.5.3/envs/borg-env/bin:/home/vagrant/.pyenv/plugins/pyenv-virtualenv/shims:/home/vagrant/.pyenv/shims:/home/vagrant/.pyenv/bin:/vagrant/borg:/home/vagrant/.pyenv/plugins/pyenv-virtualenv/shims:/home/vagrant/.pyenv/shims:/home/vagrant/.pyenv/bin:/vagrant/borg:/usr/local/bin:/bin:/usr/bin:/usr/local/sbin:/usr/sbin:/sbin:/home/vagrant/bin:/home/vagrant/bin', 'HOSTNAME': 'centos6-64', 'LD_PRELOAD': 'libfakeroot-tcp.so', 'USER': 'vagrant', 'SSH_CLIENT': '10.0.2.2 48362 22', 'HISTCONTROL': 'ignoredups', 'PYTHON_CONFIGURE_OPTS': '--enable-shared', 'SELINUX_USE_CURRENT_RANGE': '', 'PYENV_VIRTUALENV_INIT': '1', 'PYTHONHASHSEED': '3172688560', 'VIRTUAL_ENV': '/vagrant/borg/borg/.tox/py35', 'PYENV_SHELL': 'bash', 'HISTSIZE': '1000', 'PWD': '/vagrant/borg/borg', 'LESSOPEN': '||/usr/bin/lesspipe.sh %s', 'G_BROKEN_FILENAMES': '1', 'SHELL': '/bin/bash', 'FAKEROOTKEY': '38651', 'LANG': 'en_US.UTF-8', 'MAIL': '/var/spool/mail/vagrant', 'LOGNAME': 'vagrant', 'LD_LIBRARY_PATH': '/usr/lib64/libfakeroot:/usr/lib64/libfakeroot:/usr/lib32/libfakeroot', 'FAKED_MODE': 'unknown-is-real', 'SELINUX_ROLE_REQUESTED': '', 'SSH_CONNECTION': '10.0.2.2 48362 10.0.2.15 22', 'SHLVL': '4', 'SELINUX_LEVEL_REQUESTED': '', 'HOME': '/home/vagrant', 'XDISTN': '12'}
==> centos6_64: 
==> centos6_64: Processing ./.tox/dist/borgbackup-1.1.0b6.dev193+g71495b9.d20170605.zip
==> centos6_64: Requirement already satisfied: msgpack-python>=0.4.6 in ./.tox/py35/lib/python3.5/site-packages (from borgbackup==1.1.0b6.dev193+g71495b9.d20170605)
==> centos6_64: Installing collected packages: borgbackup
==> centos6_64:   Running setup.py install for borgbackup: started
==> centos6_64:     Running setup.py install for borgbackup: finished with status 'error'
==> centos6_64:     Complete output from command /vagrant/borg/borg/.tox/py35/bin/python3.5 -u -c "import setuptools, tokenize;__file__='/tmp/pip-h7m7ubu3-build/setup.py';f=getattr(tokenize, 'open', open)(__file__);code=f.read().replace('\r\n', '\n');f.close();exec(compile(code, __file__, 'exec'))" install --record /tmp/pip-wxdyxtfx-record/install-record.txt --single-version-externally-managed --compile --install-headers /vagrant/borg/borg/.tox/py35/include/site/python3.5/borgbackup:
==> centos6_64:     running install
==> centos6_64:     running build
==> centos6_64:     running build_py
==> centos6_64:     creating build
==> centos6_64:     creating build/lib.linux-x86_64-3.5
==> centos6_64:     creating build/lib.linux-x86_64-3.5/borg
==> centos6_64:     copying src/borg/cache.py -> build/lib.linux-x86_64-3.5/borg
...
==> centos6_64:     running egg_info
==> centos6_64:     writing requirements to src/borgbackup.egg-info/requires.txt
==> centos6_64:     writing src/borgbackup.egg-info/PKG-INFO
==> centos6_64:     writing dependency_links to src/borgbackup.egg-info/dependency_links.txt
==> centos6_64:     writing entry points to src/borgbackup.egg-info/entry_points.txt
==> centos6_64:     writing top-level names to src/borgbackup.egg-info/top_level.txt
==> centos6_64:     reading manifest file 'src/borgbackup.egg-info/SOURCES.txt'
==> centos6_64:     reading manifest template 'MANIFEST.in'
==> centos6_64:     writing manifest file 'src/borgbackup.egg-info/SOURCES.txt'
==> centos6_64:     copying src/borg/_hashindex.c -> build/lib.linux-x86_64-3.5/borg
...
==> centos6_64:     creating build/lib.linux-x86_64-3.5/borg/algorithms/blake2
==> centos6_64:     copying src/borg/algorithms/blake2/COPYING -> build/lib.linux-x86_64-3.5/borg/algorithms/blake2
...
==> centos6_64:     creating build/lib.linux-x86_64-3.5/borg/algorithms/xxh64
==> centos6_64:     copying src/borg/algorithms/xxh64/xxhash.c -> build/lib.linux-x86_64-3.5/borg/algorithms/xxh64
==> centos6_64:     copying src/borg/algorithms/xxh64/xxhash.h -> build/lib.linux-x86_64-3.5/borg/algorithms/xxh64
==> centos6_64:     running build_ext
==> centos6_64:     skipping 'src/borg/compress.c' Cython extension (up-to-date)
==> centos6_64:     building 'borg.compress' extension
==> centos6_64:     creating build/temp.linux-x86_64-3.5
==> centos6_64:     creating build/temp.linux-x86_64-3.5/src
==> centos6_64:     creating build/temp.linux-x86_64-3.5/src/borg
==> centos6_64:     gcc -pthread -Wsign-compare -Wunreachable-code -DNDEBUG -g -fwrapv -O3 -Wall -Wstrict-prototypes -fPIC -I/usr/include -I/usr/include -I/home/vagrant/.pyenv/versions/3.5.0/include/python3.5m -c src/borg/compress.c -o build/temp.linux-x86_64-3.5/src/borg/compress.o
==> centos6_64:     src/borg/compress.c: In function ‘__pyx_pf_4borg_8compress_10Compressor_6detect’:
==> centos6_64:     src/borg/compress.c:6906: warning: will never be executed
==> centos6_64:     src/borg/compress.c:6906: warning: will never be executed
==> centos6_64:     src/borg/compress.c:6906: warning: will never be executed
==> centos6_64:     src/borg/compress.c:6765: warning: will never be executed
==> centos6_64:     src/borg/compress.c:6757: warning: will never be executed
==> centos6_64:     src/borg/compress.c: In function ‘__Pyx_IternextUnpackEndCheck’:
==> centos6_64:     src/borg/compress.c:10514: warning: will never be executed
==> centos6_64:     src/borg/compress.c: In function ‘__Pyx_PyInt_AndObjC’:
==> centos6_64:     src/borg/compress.c:10694: warning: will never be executed
==> centos6_64:     src/borg/compress.c:10683: warning: will never be executed
==> centos6_64:     src/borg/compress.c:10679: warning: will never be executed
==> centos6_64:     src/borg/compress.c:10673: warning: will never be executed
==> centos6_64:     src/borg/compress.c:10669: warning: will never be executed
==> centos6_64:     src/borg/compress.c:10663: warning: will never be executed
==> centos6_64:     src/borg/compress.c:10659: warning: will never be executed
==> centos6_64:     src/borg/compress.c:10653: warning: will never be executed
...
==> centos6_64:     src/borg/compress.c: In function ‘__pyx_pf_4borg_8compress_14CompressorBase_4decide’:
==> centos6_64:     src/borg/compress.c:1655: warning: will never be executed
==> centos6_64:     src/borg/compress.c:1655: warning: will never be executed
==> centos6_64:     src/borg/compress.c: In function ‘__pyx_bisect_code_objects’:
==> centos6_64:     src/borg/compress.c:11931: warning: will never be executed
==> centos6_64:     src/borg/compress.c: In function ‘__Pyx_CyFunction_get_kwdefaults’:
==> centos6_64:     src/borg/compress.c:11500: warning: will never be executed
==> centos6_64:     src/borg/compress.c: In function ‘__Pyx_CyFunction_get_defaults’:
==> centos6_64:     src/borg/compress.c:11470: warning: will never be executed
==> centos6_64:     src/borg/compress.c: In function ‘__Pyx_CyFunction_set_annotations’:
==> centos6_64:     src/borg/compress.c:11510: warning: will never be executed
==> centos6_64:     src/borg/compress.c: In function ‘__Pyx_CyFunction_set_kwdefaults’:
==> centos6_64:     src/borg/compress.c:11480: warning: will never be executed
==> centos6_64:     src/borg/compress.c: In function ‘__Pyx_CyFunction_set_defaults’:
==> centos6_64:     src/borg/compress.c:11450: warning: will never be executed
...
==> centos6_64:     gcc -pthread -Wsign-compare -Wunreachable-code -DNDEBUG -g -fwrapv -O3 -Wall -Wstrict-prototypes -fPIC -I/usr/include -I/usr/include -I/home/vagrant/.pyenv/versions/3.5.0/include/python3.5m -c src/borg/crypto/low_level.c -o build/temp.linux-x86_64-3.5/src/borg/crypto/low_level.o
==> centos6_64:     src/borg/crypto/low_level.c: In function ‘__Pyx_PyInt_From_PY_LONG_LONG’:
==> centos6_64:     src/borg/crypto/low_level.c:8357: warning: will never be executed
==> centos6_64:     src/borg/crypto/low_level.c:8352: warning: will never be executed
==> centos6_64:     src/borg/crypto/low_level.c:8344: warning: will never be executed
==> centos6_64:     src/borg/crypto/low_level.c:8341: warning: will never be executed
...
==> centos6_64:     src/borg/crypto/low_level.c:2385: warning: ‘__pyx_r$buf’ may be used uninitialized in this function
==> centos6_64:     src/borg/crypto/low_level.c:2385: note: ‘__pyx_r$buf’ was declared here
==> centos6_64:     src/borg/crypto/low_level.c:2385: warning: ‘__pyx_r$obj’ may be used uninitialized in this function
==> centos6_64:     src/borg/crypto/low_level.c:2385: note: ‘__pyx_r$obj’ was declared here
==> centos6_64:     src/borg/crypto/low_level.c:2385: warning: ‘__pyx_r$len’ may be used uninitialized in this function
==> centos6_64:     src/borg/crypto/low_level.c:2385: note: ‘__pyx_r$len’ was declared here
==> centos6_64:     src/borg/crypto/low_level.c:2385: warning: ‘__pyx_r$itemsize’ may be used uninitialized in this function
==> centos6_64:     src/borg/crypto/low_level.c:2385: note: ‘__pyx_r$itemsize’ was declared here
==> centos6_64:     src/borg/crypto/low_level.c:2385: warning: ‘__pyx_r$format’ may be used uninitialized in this function
==> centos6_64:     src/borg/crypto/low_level.c:2385: note: ‘__pyx_r$format’ was declared here
==> centos6_64:     src/borg/crypto/low_level.c:2385: warning: ‘__pyx_r$shape’ may be used uninitialized in this function
==> centos6_64:     src/borg/crypto/low_level.c:2385: note: ‘__pyx_r$shape’ was declared here
==> centos6_64:     src/borg/crypto/low_level.c:2385: warning: ‘__pyx_r$strides’ may be used uninitialized in this function
==> centos6_64:     src/borg/crypto/low_level.c:2385: note: ‘__pyx_r$strides’ was declared here
==> centos6_64:     src/borg/crypto/low_level.c:2385: warning: ‘__pyx_r$suboffsets’ may be used uninitialized in this function
==> centos6_64:     src/borg/crypto/low_level.c:2385: note: ‘__pyx_r$suboffsets’ was declared here
==> centos6_64:     src/borg/crypto/low_level.c:2385: warning: ‘__pyx_r$internal’ may be used uninitialized in this function
==> centos6_64:     src/borg/crypto/low_level.c:2385: note: ‘__pyx_r$internal’ was declared here
...
...
==> centos6_64:     src/borg/hashindex.c:9116: warning: will never be executed
==> centos6_64:     src/borg/hashindex.c:9116: warning: will never be executed
==> centos6_64:     src/borg/hashindex.c:9113: warning: will never be executed
==> centos6_64:     src/borg/hashindex.c:9113: warning: will never be executed
==> centos6_64:     src/borg/hashindex.c:9113: warning: will never be executed
==> centos6_64:     src/borg/hashindex.c:9113: warning: will never be executed
==> centos6_64:     src/borg/hashindex.c:9113: warning: will never be executed
==> centos6_64:     src/borg/hashindex.c:9113: warning: will never be executed
...
==> centos6_64:     src/borg/hashindex.c:1584: warning: will never be executed
==> centos6_64:     src/borg/hashindex.c:1587: warning: will never be executed
==> centos6_64:     src/borg/hashindex.c: In function ‘__pyx_tp_new_4borg_9hashindex_ChunkIndex’:
==> centos6_64:     src/borg/hashindex.c:6688: warning: will never be executed
==> centos6_64:     gcc -pthread -shared -L/home/vagrant/.pyenv/versions/3.5.0/lib -Wl,-rpath=/home/vagrant/.pyenv/versions/3.5.0/lib build/temp.linux-x86_64-3.5/src/borg/hashindex.o -L/home/vagrant/.pyenv/versions/3.5.0/lib -lpython3.5m -o build/lib.linux-x86_64-3.5/borg/hashindex.cpython-35m-x86_64-linux-gnu.so
==> centos6_64:     skipping 'src/borg/item.c' Cython extension (up-to-date)
==> centos6_64:     building 'borg.item' extension
==> centos6_64:     gcc -pthread -Wsign-compare -Wunreachable-code -DNDEBUG -g -fwrapv -O3 -Wall -Wstrict-prototypes -fPIC -I/home/vagrant/.pyenv/versions/3.5.0/include/python3.5m -c src/borg/item.c -o build/temp.linux-x86_64-3.5/src/borg/item.o
==> centos6_64:     src/borg/item.c: In function ‘__Pyx_IternextUnpackEndCheck’:
==> centos6_64:     src/borg/item.c:10908: warning: will never be executed
==> centos6_64:     src/borg/item.c: In function ‘__pyx_gb_4borg_4item_4Item_8get_size_2generator’:
==> centos6_64:     src/borg/item.c:4570: warning: will never be executed
==> centos6_64:     src/borg/item.c:4562: warning: will never be executed
==> centos6_64:     src/borg/item.c: In function ‘__pyx_pf_4borg_4item_8PropDict_4update_internal’:
...
==> centos6_64:     src/borg/item.c:3032: warning: will never be executed
==> centos6_64:     gcc -pthread -shared -L/home/vagrant/.pyenv/versions/3.5.0/lib -Wl,-rpath=/home/vagrant/.pyenv/versions/3.5.0/lib build/temp.linux-x86_64-3.5/src/borg/item.o -L/home/vagrant/.pyenv/versions/3.5.0/lib -lpython3.5m -o build/lib.linux-x86_64-3.5/borg/item.cpython-35m-x86_64-linux-gnu.so
==> centos6_64:     skipping 'src/borg/algorithms/chunker.c' Cython extension (up-to-date)
==> centos6_64:     building 'borg.algorithms.chunker' extension
==> centos6_64:     creating build/temp.linux-x86_64-3.5/src/borg/algorithms
==> centos6_64:     gcc -pthread -Wsign-compare -Wunreachable-code -DNDEBUG -g -fwrapv -O3 -Wall -Wstrict-prototypes -fPIC -I/home/vagrant/.pyenv/versions/3.5.0/include/python3.5m -c src/borg/algorithms/chunker.c -o build/temp.linux-x86_64-3.5/src/borg/algorithms/chunker.o
==> centos6_64:     src/borg/algorithms/chunker.c: In function ‘__Pyx_PyInt_As_int’:
==> centos6_64:     src/borg/algorithms/chunker.c:2797: warning: will never be executed
==> centos6_64:     src/borg/algorithms/chunker.c:2782: warning: will never be executed
==> centos6_64:     src/borg/algorithms/chunker.c:2779: warning: will never be executed
...
==> centos6_64:     src/borg/algorithms/chunker.c:1153: warning: will never be executed
==> centos6_64:     gcc -pthread -shared -L/home/vagrant/.pyenv/versions/3.5.0/lib -Wl,-rpath=/home/vagrant/.pyenv/versions/3.5.0/lib build/temp.linux-x86_64-3.5/src/borg/algorithms/chunker.o -L/home/vagrant/.pyenv/versions/3.5.0/lib -lpython3.5m -o build/lib.linux-x86_64-3.5/borg/algorithms/chunker.cpython-35m-x86_64-linux-gnu.so
==> centos6_64:     skipping 'src/borg/algorithms/checksums.c' Cython extension (up-to-date)
==> centos6_64:     building 'borg.algorithms.checksums' extension
==> centos6_64:     gcc -pthread -Wsign-compare -Wunreachable-code -DNDEBUG -g -fwrapv -O3 -Wall -Wstrict-prototypes -fPIC -I/home/vagrant/.pyenv/versions/3.5.0/include/python3.5m -c src/borg/algorithms/checksums.c -o build/temp.linux-x86_64-3.5/src/borg/algorithms/checksums.o
==> centos6_64:     gcc: Internal error: Segmentation fault (program cc1)
==> centos6_64:     Please submit a full bug report.
==> centos6_64:     See <http://bugzilla.redhat.com/bugzilla> for instructions.
==> centos6_64:     error: command 'gcc' failed with exit status 1

==> centos6_64:     
==> centos6_64:     ----------------------------------------
==> centos6_64: Command "/vagrant/borg/borg/.tox/py35/bin/python3.5 -u -c "import setuptools, tokenize;__file__='/tmp/pip-h7m7ubu3-build/setup.py';f=getattr(tokenize, 'open', open)(__file__);code=f.read().replace('\r\n', '\n');f.close();exec(compile(code, __file__, 'exec'))" install --record /tmp/pip-wxdyxtfx-record/install-record.txt --single-version-externally-managed --compile --install-headers /vagrant/borg/borg/.tox/py35/include/site/python3.5/borgbackup" failed with error code 1 in /tmp/pip-h7m7ubu3-build/
==> centos6_64: py35 installed: The directory '/home/vagrant/.cache/pip/http' or its parent directory is not owned by the current user and the cache has been disabled. Please check the permissions and owner of that directory. If executing pip with sudo, you may want sudo's -H flag.,apipkg==1.4,Attic==0.16,coverage==4.4.1,Cython==0.25.2,execnet==1.4.1,msgpack-python==0.4.8,pluggy==0.4.0,py==1.4.34,pytest==3.1.1,pytest-benchmark==3.0.0,pytest-cov==2.5.1,pytest-xdist==1.16.0,setuptools-scm==1.15.5,tox==2.7.0,virtualenv==15.1.0
==> centos6_64: py36 recreate: /vagrant/borg/borg/.tox/py36
==> centos6_64: py36 installdeps: -rrequirements.d/development.txt, -rrequirements.d/attic.txt
==> centos6_64: py36 inst: /vagrant/borg/borg/.tox/dist/borgbackup-1.1.0b6.dev193+g71495b9.d20170605.zip
==> centos6_64: ERROR: invocation failed (exit code 1), logfile: /vagrant/borg/borg/.tox/py36/log/py36-2.log
==> centos6_64: ERROR: actionid: py36
==> centos6_64: msg: installpkg
==> centos6_64: cmdargs: ['/vagrant/borg/borg/.tox/py36/bin/pip', 'install', '/vagrant/borg/borg/.tox/dist/borgbackup-1.1.0b6.dev193+g71495b9.d20170605.zip']
==> centos6_64: env: {'_': '/home/vagrant/.pyenv/versions/3.5.3/envs/borg-env/bin/tox', 'TERM': 'vt100', 'PATH': '/vagrant/borg/borg/.tox/py36/bin:/home/vagrant/.pyenv/versions/3.5.3/envs/borg-env/bin:/home/vagrant/.pyenv/plugins/pyenv-virtualenv/shims:/home/vagrant/.pyenv/shims:/home/vagrant/.pyenv/bin:/vagrant/borg:/home/vagrant/.pyenv/plugins/pyenv-virtualenv/shims:/home/vagrant/.pyenv/shims:/home/vagrant/.pyenv/bin:/vagrant/borg:/usr/local/bin:/bin:/usr/bin:/usr/local/sbin:/usr/sbin:/sbin:/home/vagrant/bin:/home/vagrant/bin', 'HOSTNAME': 'centos6-64', 'LD_PRELOAD': 'libfakeroot-tcp.so', 'USER': 'vagrant', 'SSH_CLIENT': '10.0.2.2 48362 22', 'HISTCONTROL': 'ignoredups', 'PYTHON_CONFIGURE_OPTS': '--enable-shared', 'SELINUX_USE_CURRENT_RANGE': '', 'PYENV_VIRTUALENV_INIT': '1', 'PYTHONHASHSEED': '3172688560', 'VIRTUAL_ENV': '/vagrant/borg/borg/.tox/py36', 'PYENV_SHELL': 'bash', 'HISTSIZE': '1000', 'PWD': '/vagrant/borg/borg', 'LESSOPEN': '||/usr/bin/lesspipe.sh %s', 'G_BROKEN_FILENAMES': '1', 'SHELL': '/bin/bash', 'FAKEROOTKEY': '38651', 'LANG': 'en_US.UTF-8', 'MAIL': '/var/spool/mail/vagrant', 'LOGNAME': 'vagrant', 'LD_LIBRARY_PATH': '/usr/lib64/libfakeroot:/usr/lib64/libfakeroot:/usr/lib32/libfakeroot', 'FAKED_MODE': 'unknown-is-real', 'SELINUX_ROLE_REQUESTED': '', 'SSH_CONNECTION': '10.0.2.2 48362 10.0.2.15 22', 'SHLVL': '4', 'SELINUX_LEVEL_REQUESTED': '', 'HOME': '/home/vagrant', 'XDISTN': '12'}
==> centos6_64: 
==> centos6_64: The directory '/home/vagrant/.cache/pip/http' or its parent directory is not owned by the current user and the cache has been disabled. Please check the permissions and owner of that directory. If executing pip with sudo, you may want sudo's -H flag.
==> centos6_64: The directory '/home/vagrant/.cache/pip' or its parent directory is not owned by the current user and caching wheels has been disabled. check the permissions and owner of that directory. If executing pip with sudo, you may want sudo's -H flag.
==> centos6_64: Processing ./.tox/dist/borgbackup-1.1.0b6.dev193+g71495b9.d20170605.zip
==> centos6_64: Requirement already satisfied: msgpack-python>=0.4.6 in ./.tox/py36/lib/python3.6/site-packages (from borgbackup==1.1.0b6.dev193+g71495b9.d20170605)
==> centos6_64: Installing collected packages: borgbackup
==> centos6_64:   Running setup.py install for borgbackup: started
==> centos6_64:     Running setup.py install for borgbackup: finished with status 'error'
==> centos6_64:     Complete output from command /vagrant/borg/borg/.tox/py36/bin/python3.6 -u -c "import setuptools, tokenize;__file__='/tmp/pip-ntg3y97o-build/setup.py';f=getattr(tokenize, 'open', open)(__file__);code=f.read().replace('\r\n', '\n');f.close();exec(compile(code, __file__, 'exec'))" install --record /tmp/pip-v6kf5t7f-record/install-record.txt --single-version-externally-managed --compile --install-headers /vagrant/borg/borg/.tox/py36/include/site/python3.6/borgbackup:
==> centos6_64:     running install
==> centos6_64:     running build
==> centos6_64:     running build_py
==> centos6_64:     creating build
==> centos6_64:     creating build/lib.linux-x86_64-3.6
==> centos6_64:     creating build/lib.linux-x86_64-3.6/borg
==> centos6_64:     copying src/borg/cache.py -> build/lib.linux-x86_64-3.6/borg
...
==> centos6_64:     creating build/lib.linux-x86_64-3.6/borg/algorithms
==> centos6_64:     copying src/borg/algorithms/__init__.py -> build/lib.linux-x86_64-3.6/borg/algorithms
==> centos6_64:     running egg_info
==> centos6_64:     writing src/borgbackup.egg-info/PKG-INFO
==> centos6_64:     writing dependency_links to src/borgbackup.egg-info/dependency_links.txt
==> centos6_64:     writing entry points to src/borgbackup.egg-info/entry_points.txt
==> centos6_64:     writing requirements to src/borgbackup.egg-info/requires.txt
==> centos6_64:     writing top-level names to src/borgbackup.egg-info/top_level.txt
==> centos6_64:     reading manifest file 'src/borgbackup.egg-info/SOURCES.txt'
==> centos6_64:     reading manifest template 'MANIFEST.in'
==> centos6_64:     writing manifest file 'src/borgbackup.egg-info/SOURCES.txt'
==> centos6_64:     copying src/borg/_hashindex.c -> build/lib.linux-x86_64-3.6/borg
...
==> centos6_64:     creating build/lib.linux-x86_64-3.6/borg/algorithms/blake2
==> centos6_64:     copying src/borg/algorithms/blake2/COPYING -> build/lib.linux-x86_64-3.6/borg/algorithms/blake2
...
==> centos6_64:     creating build/lib.linux-x86_64-3.6/borg/algorithms/xxh64
==> centos6_64:     copying src/borg/algorithms/xxh64/xxhash.c -> build/lib.linux-x86_64-3.6/borg/algorithms/xxh64
==> centos6_64:     copying src/borg/algorithms/xxh64/xxhash.h -> build/lib.linux-x86_64-3.6/borg/algorithms/xxh64
==> centos6_64:     running build_ext
==> centos6_64:     skipping 'src/borg/compress.c' Cython extension (up-to-date)
==> centos6_64:     building 'borg.compress' extension
==> centos6_64:     creating build/temp.linux-x86_64-3.6
==> centos6_64:     creating build/temp.linux-x86_64-3.6/src
==> centos6_64:     creating build/temp.linux-x86_64-3.6/src/borg
==> centos6_64:     gcc -pthread -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall -Wstrict-prototypes -fPIC -I/usr/include -I/usr/include -I/home/vagrant/.pyenv/versions/3.6.0/include/python3.6m -c src/borg/compress.c -o build/temp.linux-x86_64-3.6/src/borg/compress.o
==> centos6_64:     gcc -pthread -shared -L/home/vagrant/.pyenv/versions/3.6.0/lib -Wl,-rpath=/home/vagrant/.pyenv/versions/3.6.0/lib -L/home/vagrant/.pyenv/versions/3.6.0/lib -Wl,-rpath=/home/vagrant/.pyenv/versions/3.6.0/lib build/temp.linux-x86_64-3.6/src/borg/compress.o -L/usr/lib -L/usr/lib -L/home/vagrant/.pyenv/versions/3.6.0/lib -llz4 -lpython3.6m -o build/lib.linux-x86_64-3.6/borg/compress.cpython-36m-x86_64-linux-gnu.so
==> centos6_64:     skipping 'src/borg/crypto/low_level.c' Cython extension (up-to-date)
==> centos6_64:     building 'borg.crypto.low_level' extension
==> centos6_64:     creating build/temp.linux-x86_64-3.6/src/borg/crypto
==> centos6_64:     gcc -pthread -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall -Wstrict-prototypes -fPIC -I/usr/include -I/usr/include -I/home/vagrant/.pyenv/versions/3.6.0/include/python3.6m -c src/borg/crypto/low_level.c -o build/temp.linux-x86_64-3.6/src/borg/crypto/low_level.o
==> centos6_64:     src/borg/crypto/low_level.c: In function ‘__pyx_pf_4borg_6crypto_9low_level_3AES_8decrypt’:
==> centos6_64:     src/borg/crypto/low_level.c:2385: warning: ‘__pyx_r$buf’ may be used uninitialized in this function
==> centos6_64:     src/borg/crypto/low_level.c:2385: note: ‘__pyx_r$buf’ was declared here
==> centos6_64:     src/borg/crypto/low_level.c:2385: warning: ‘__pyx_r$obj’ may be used uninitialized in this function
==> centos6_64:     src/borg/crypto/low_level.c:2385: note: ‘__pyx_r$obj’ was declared here
==> centos6_64:     src/borg/crypto/low_level.c:2385: warning: ‘__pyx_r$len’ may be used uninitialized in this function
==> centos6_64:     src/borg/crypto/low_level.c:2385: note: ‘__pyx_r$len’ was declared here
==> centos6_64:     src/borg/crypto/low_level.c:2385: warning: ‘__pyx_r$itemsize’ may be used uninitialized in this function
...
==> centos6_64:     src/borg/crypto/low_level.c:2385: warning: ‘__pyx_r$internal’ may be used uninitialized in this function
==> centos6_64:     src/borg/crypto/low_level.c:2385: note: ‘__pyx_r$internal’ was declared here
==> centos6_64:     gcc -pthread -shared -L/home/vagrant/.pyenv/versions/3.6.0/lib -Wl,-rpath=/home/vagrant/.pyenv/versions/3.6.0/lib -L/home/vagrant/.pyenv/versions/3.6.0/lib -Wl,-rpath=/home/vagrant/.pyenv/versions/3.6.0/lib build/temp.linux-x86_64-3.6/src/borg/crypto/low_level.o -L/usr/lib -L/usr/lib -L/home/vagrant/.pyenv/versions/3.6.0/lib -lcrypto -lpython3.6m -o build/lib.linux-x86_64-3.6/borg/crypto/low_level.cpython-36m-x86_64-linux-gnu.so
==> centos6_64:     skipping 'src/borg/hashindex.c' Cython extension (up-to-date)
==> centos6_64:     building 'borg.hashindex' extension
==> centos6_64:     gcc -pthread -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall -Wstrict-prototypes -fPIC -I/home/vagrant/.pyenv/versions/3.6.0/include/python3.6m -c src/borg/hashindex.c -o build/temp.linux-x86_64-3.6/src/borg/hashindex.o
==> centos6_64:     In file included from src/borg/hashindex.c:436:
==> centos6_64:     src/borg/_hashindex.c: In function ‘hashindex_set’:
==> centos6_64:     src/borg/_hashindex.c:538: warning: ‘start_idx’ may be used uninitialized in this function
==> centos6_64:     gcc -pthread -shared -L/home/vagrant/.pyenv/versions/3.6.0/lib -Wl,-rpath=/home/vagrant/.pyenv/versions/3.6.0/lib -L/home/vagrant/.pyenv/versions/3.6.0/lib -Wl,-rpath=/home/vagrant/.pyenv/versions/3.6.0/lib build/temp.linux-x86_64-3.6/src/borg/hashindex.o -L/home/vagrant/.pyenv/versions/3.6.0/lib -lpython3.6m -o build/lib.linux-x86_64-3.6/borg/hashindex.cpython-36m-x86_64-linux-gnu.so
==> centos6_64:     skipping 'src/borg/item.c' Cython extension (up-to-date)
==> centos6_64:     building 'borg.item' extension
==> centos6_64:     gcc -pthread -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall -Wstrict-prototypes -fPIC -I/home/vagrant/.pyenv/versions/3.6.0/include/python3.6m -c src/borg/item.c -o build/temp.linux-x86_64-3.6/src/borg/item.o
==> centos6_64:     gcc -pthread -shared -L/home/vagrant/.pyenv/versions/3.6.0/lib -Wl,-rpath=/home/vagrant/.pyenv/versions/3.6.0/lib -L/home/vagrant/.pyenv/versions/3.6.0/lib -Wl,-rpath=/home/vagrant/.pyenv/versions/3.6.0/lib build/temp.linux-x86_64-3.6/src/borg/item.o -L/home/vagrant/.pyenv/versions/3.6.0/lib -lpython3.6m -o build/lib.linux-x86_64-3.6/borg/item.cpython-36m-x86_64-linux-gnu.so
==> centos6_64:     skipping 'src/borg/algorithms/chunker.c' Cython extension (up-to-date)
==> centos6_64:     building 'borg.algorithms.chunker' extension
==> centos6_64:     creating build/temp.linux-x86_64-3.6/src/borg/algorithms
==> centos6_64:     gcc -pthread -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall -Wstrict-prototypes -fPIC -I/home/vagrant/.pyenv/versions/3.6.0/include/python3.6m -c src/borg/algorithms/chunker.c -o build/temp.linux-x86_64-3.6/src/borg/algorithms/chunker.o
==> centos6_64:     gcc -pthread -shared -L/home/vagrant/.pyenv/versions/3.6.0/lib -Wl,-rpath=/home/vagrant/.pyenv/versions/3.6.0/lib -L/home/vagrant/.pyenv/versions/3.6.0/lib -Wl,-rpath=/home/vagrant/.pyenv/versions/3.6.0/lib build/temp.linux-x86_64-3.6/src/borg/algorithms/chunker.o -L/home/vagrant/.pyenv/versions/3.6.0/lib -lpython3.6m -o build/lib.linux-x86_64-3.6/borg/algorithms/chunker.cpython-36m-x86_64-linux-gnu.so
==> centos6_64:     skipping 'src/borg/algorithms/checksums.c' Cython extension (up-to-date)
==> centos6_64:     building 'borg.algorithms.checksums' extension
==> centos6_64:     gcc -pthread -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall -Wstrict-prototypes -fPIC -I/home/vagrant/.pyenv/versions/3.6.0/include/python3.6m -c src/borg/algorithms/checksums.c -o build/temp.linux-x86_64-3.6/src/borg/algorithms/checksums.o
==> centos6_64:     gcc: Internal error: Segmentation fault (program cc1)
==> centos6_64:     Please submit a full bug report.
==> centos6_64:     See <http://bugzilla.redhat.com/bugzilla> for instructions.
==> centos6_64:     error: command 'gcc' failed with exit status 1

==> centos6_64:     
==> centos6_64:     ----------------------------------------
==> centos6_64: Command "/vagrant/borg/borg/.tox/py36/bin/python3.6 -u -c "import setuptools, tokenize;__file__='/tmp/pip-ntg3y97o-build/setup.py';f=getattr(tokenize, 'open', open)(__file__);code=f.read().replace('\r\n', '\n');f.close();exec(compile(code, __file__, 'exec'))" install --record /tmp/pip-v6kf5t7f-record/install-record.txt --single-version-externally-managed --compile --install-headers /vagrant/borg/borg/.tox/py36/include/site/python3.6/borgbackup" failed with error code 1 in /tmp/pip-ntg3y97o-build/
```
Besides the segfault, there is also an enormous amount of warnings (I deleted most of the similar ones).

--- a/src/borg/algorithms/crc32_dispatch.c
+++ b/src/borg/algorithms/crc32_dispatch.c
@@ -4,6 +4,12 @@
 
 #ifdef __GNUC__
 /*
+ * GCC 4.4(.7) has a bug that causes it to recurse infinitely if an unknown option
+ * is pushed onto the options stack. GCC 4.5 was not tested, so is excluded as well.
+ * GCC 4.6 is known good.
+ */
+#if __GNUC__ > 4 || (__GNUC__ == 4 && __GNUC_MINOR__ >= 6)
+/*
  * clang also has or had GCC bug #56298 explained below, but doesn't support
  * target attributes or the options stack. So we disable this faster code path for clang.
  */
@@ -66,6 +72,7 @@
 #endif /* if __x86_64__ */
 #endif /* ifndef __OpenBSD__ */
 #endif /* ifndef __clang__ */
+#endif /* __GNUC__ > 4 || (__GNUC__ == 4 && __GNUC_MINOR__ >= 6) */
 #endif /* ifdef __GNUC__ */
 
 #ifdef FOLDING_CRC
@@ -75,6 +82,7 @@
 static uint32_t
 crc32_clmul(const uint8_t *src, long len, uint32_t initial_crc)
 {
+    (void)src; (void)len; (void)initial_crc;
     assert(0);
     return 0;
 }

